code/pizzabot.js

```diff
@@ -1,7 +1,53 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
 
 //Put your Javscript code here:
+
+//test
+//console.log(`Test if connection HTML-JS works: ${vegetarian}, ${hawaiian} and ${pepperoni}`);
+
+alert(`Welcome, customer! Happy to serve your pizza. On our menu we have: ${vegetarian}, ${hawaiian} and ${pepperoni}`);
+
+const orderName = prompt(`Enter the name of the pizza you want to order today:`);
+
+//Function that tests whether the user input matches the pizza name
+const validateOrderName = (orderName) => {
+    if (orderName.toLowerCase() === vegetarian.toLowerCase() ||
+    orderName.toLowerCase() === hawaiian.toLowerCase() ||
+    orderName.toLowerCase() === pepperoni.toLowerCase ()) {
+        return true;
+    }
+    else {
+        return false;
+    }
+};
+
+var orderQuantity;
+var orderTotal;
+
+if (validateOrderName(orderName)) {
+    orderQuantity = prompt(`How many of ${orderName}s do you want?`);
+    orderTotal = orderQuantity * pizzaPrice;
+    alert(`Great, I'll get started on your ${orderName}s right away, it will cost ${orderTotal} kr.`);
```

Comment:
This should be broken out into another function, called something like `calculateTotalCost()` or `calculateOrderTotal()`